docs/content/en/docs/core-components/api-manual.md

```diff
@@ -0,0 +1,223 @@
+|out_args          |cacao variables        |dictionary         |Map of [cacao variables](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256555) handled in the step out args with current values and definitions
+
+
+
+```plantuml
+@startjson
+
+    {
+        "type" :        "manual-step-information",
+        "execution_id" : "<execution-id>",
+        "playbook_id" :  "<playbook-id>",
+        "step_id" :  "<step-id>",
+        "command" : "<command here>",
+        "command_is_base64" : "false",
+        "targets" : {
+            "__target1__" : {
+                "type" : "<agent-target-type-ov>",
+                "name" : "<agent name>",
+                "description" : "<some description>",
+                "location" : "<.>",
+                "agent_target_extensions" : {}
+            }
+        },
+        "in_args":    {
+            "<variable-name-1>" : {
+                "type":         "<type>",
+                "name":         "<variable-name>",
+                "description":  "<description>",
+                "value":        "<value>",
+                "constant":     "<true/false>",
+                "external":     "<true/false>"
+            }
+        },
+        "out_args":    {
+            "<variable-name-1>" : {
+                "type":         "<type>",
+                "name":         "<variable-name>",
+                "description":  "<description>",
+                "value":        "<value>",
+                "constant":     "<true/false>",
+                "external":     "<true/false>"
+            }
+        }
+    }
+
+@endjson
+```
+
+##### Error
+404/Not found with payload:
+General error
+
+#### POST `/manual/continue`
+Respond to manual command pending in SOARCA, if out_args are defined they must be filled in and returned in the payload body
+
+##### Call payload
+|field              |content                |type               | description |
+| ----------------- | --------------------- | ----------------- | ----------- |
+|type               |execution-status       |string             |The type of this content
+|execution_id       |UUID                   |string             |The id of the execution
+|playbook_id        |UUID                   |string             |The id of the CACAO playbook executed by the execution
+|step_id            |UUID                   |string             |The id of the step executed by the execution
+|response_status    |enum                   |string             |Can be either `success` or `failed`
+|response_out_args  |cacao variables        |dictionary         |Map of [cacao variables](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256555) handled in the step out args with current values and definitions
```

Comment:
I am thinking whether we should also expect, as part of the out_arg info, anything else besides the arg (variable) name, and its value. 
Perhaps it may make sense to allow passing back `type`, and `description` if we can have a wildcard for it (but would we want a wildcard variable? That could mean inconsistent executions? Or else imply that a wildcard variable is properly handled in a playbook?). But `constant` and `external`? I am not sure